packages/webview_flutter/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -95,6 +95,20 @@ - (instancetype)initWithFrame:(CGRect)frame
                         inConfiguration:configuration];
 
     _webView = [[FLTWKWebView alloc] initWithFrame:frame configuration:configuration];
+
+    // Background color
+    NSNumber* backgroundColorNSNumber = args[@"backgroundColor"];
+    if ([backgroundColorNSNumber isKindOfClass:[NSNumber class]]) {
+      int backgroundColorInt = [backgroundColorNSNumber intValue];
+      UIColor* backgroundColor = [UIColor colorWithRed:(backgroundColorInt >> 16 & 0xff) / 255.f
+                                                 green:(backgroundColorInt >> 8 & 0xff) / 255.f
+                                                  blue:(backgroundColorInt & 0xff) / 255.f
+                                                 alpha:(backgroundColorInt >> 24 & 0xff) / 255.f];
+      _webView.opaque = NO;
+      _webView.backgroundColor = backgroundColor;
+      _webView.scrollView.backgroundColor = backgroundColor;
```

Comment:
Does setting both of these actually work with semi-opaque values? I would expect it to layer and give the wrong value.

It would be good to have a test covering that.